### ISSUE TITLE
[KEYBOARD]: Profile 2.0 - Focus MUST be managed when users switch from the read-only to edit in place UI

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#11965]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Each of the profile "buckets" of information include an Edit button that lets users edit in place. When users click "Edit", focus is lost to the `<body>` tag. Modern browsers like Chrome and Firefox remember where focus was before it was lost, but IE11 forgets and moves focus to the end of the document. This forces IE users to `SHIFT + TAB` back through the document to edit their content. Screenshot attached below shows the moment when focus is lost.

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/4613 **(blocks this issue)**

## Point of Contact

**VFS Point of Contact:** _Jennifer, Trevor_

## Acceptance Criteria

- [ ] Focus is moved to the first native focusable element when users press "Edit"
- [ ] Focus is returned to the "Edit" button when users update or cancel their edit in place

## WCAG or Vendor Guidance (optional)

* [Focus Order: Understanding SC 2.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html)

## Screenshots or Trace Logs

![Screen Shot 2020-08-03 at 3 18 22 PM](https://user-images.githubusercontent.com/934879/89224927-f595b680-d59e-11ea-9d42-23a83a65de79.png)

**Comment by taylorkaren** [comment]
Info from @micahchiang:  The edit in place pattern is something we probably want to provide eventually, but it isn't a priority for the Forms Library team during Q2.

**Comment by taylorkaren** [comment]
Description of this ticket indicates that it is only an issue in IE11. Review this with @humancompanion-usds; given that IE11 is no longer supported by Microsoft, we may be able to close this issue with no further action.

**Comment by taylorkaren** [comment]
@obliviga: Per Matt, this is in MyVA/Profile thus Liz Lantz on that team should be able to direct you to the right page. Looks like from the screenshot it is when you edit your address. Should be able to get to that on staging once authenticated.
